SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -70,60 +70,41 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
                 WorkerLibraryDir,
             };
 
-        string LibPrefix = "improbable_";
-        string ImportLibSuffix = "";
-        string SharedLibSuffix = "";
+        string LibPrefix = "libimprobable_";
+        string ImportLibSuffix = ".so";
+        string SharedLibSuffix = ".so";
         bool bAddDelayLoad = false;
 
         if (Target.Platform == UnrealTargetPlatform.Win32 || Target.Platform == UnrealTargetPlatform.Win64)
         {
+            LibPrefix = "improbable_";
             ImportLibSuffix = ".lib";
             SharedLibSuffix = ".dll";
             bAddDelayLoad = true;
         }
         else if (Target.Platform == UnrealTargetPlatform.Mac)
         {
-            LibPrefix = "libimprobable_";
             ImportLibSuffix = SharedLibSuffix = ".dylib";
         }
-        else if (Target.Platform == UnrealTargetPlatform.Linux)
-        {
-            LibPrefix = "libimprobable_";
-            ImportLibSuffix = SharedLibSuffix = ".so";
-        }
         else if (Target.Platform == UnrealTargetPlatform.PS4)
         {
-            LibPrefix = "libimprobable_";
             ImportLibSuffix = "_stub.a";
             SharedLibSuffix = ".prx";
             bAddDelayLoad = true;
         }
         else if (Target.Platform == UnrealTargetPlatform.XboxOne)
         {
+            LibPrefix = "improbable_";
             ImportLibSuffix = ".lib";
             SharedLibSuffix = ".dll";
             // We don't set bAddDelayLoad = true here, because we get "unresolved external symbol __delayLoadHelper2".
             // See: https://www.fmod.org/questions/question/deploy-issue-on-xboxone-with-unrealengine-4-14/
         }
         else if (Target.Platform == UnrealTargetPlatform.IOS)
         {
-            LibPrefix = "libimprobable_";
             ImportLibSuffix = SharedLibSuffix = "_static.a";
         }
-        else if (Target.Platform == UnrealTargetPlatform.Android)
-        {
-            LibPrefix = "improbable_";
```

Comment:
Looks like we now use "libimprobable_" for android. Is this a fix?

Reply:
Android libs are named `libimprobable_worker.so`. It's unclear why we were previously passing just `improbable_worker`, or why that worked.